mix.exs

```diff
@@ -49,15 +49,15 @@ defmodule Wallaby.Mixfile do
 
   defp deps do
     [
-      {:httpoison, "~> 0.12"},
+      {:httpoison, "~> 0.13"},
```

Comment:
Is there a feature we need to use from the new httpoison? For runtime requirements I'd always try to keep it as the lowest common denominator, I mean we allow 0.13 already with this :)

Reply:
I don't believe there's anything in there that we need. We can keep it at 0.12 for a while longer.